### ISSUE TITLE
Add support for custom gtm.js filename

### ISSUE THREAD

**Issue by simkoG** [duracelltomi/gtm4wp#230]
You could consider adding support for the custom `gtm.js` filename alongside with custom domain to prevent adblockers to block the file. 

For example, if I make a GTM container server with [stape.io](https://stape.io/), it automatically renames the `gtm.js` file. With your plugin, I have to turn off the "Container code ON/OFF" option, and manually insert the GTM code because, with this server, the `gtm.js` file is unreachable. 

![ublock-debugger](https://user-images.githubusercontent.com/10430702/175511378-eed92ecf-a962-4275-af4c-d1f75ea0ac91.png)

**Comment by simkoG** [comment]
Also, you could remove this console warning message if the GTM container code was added. 

![image](https://user-images.githubusercontent.com/10430702/175524635-c8fb695d-d819-4909-acf7-c35d0f059956.png)

**Comment by duracelltomi** [comment]
Hi,

I've added this to the TODO list.

The console commands help me and users a lot to see reasons for possible issues. I had countless cases where such a message could helped me to understand the issue more quickly. But if that is a good compromise, I could add a "Debug mode" checkbox that can be turned on upon request.

**Comment by simkoG** [comment]
Dear @duracelltomi, I've found another bug while trying to install my GTM container.

Your plugin has a `GTM4WP_WPACTION_AFTER_DATALAYER` / `gtm4wp_output_after_datalayer` hook to load some custom code after the `dataLayer` initaliazition code. I tried to insert the GTM js code into my page by calling `add_action( 'gtm4wp_output_after_datalayer', 'acme_my_function' );`.

The problem is that this hook is called twice per page. First in the head (after the dataLayer section). This is fine. 
But the problem is, that this hook is called again after the <body> tag, so my Tag Manager code loads twice per page. 

(For the `<noscript>` tag I've used `wp_body_open` hook, this is working fine.)
